### ISSUE TITLE
Accessibility: Element ID is not unique

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#263]
Hello! On our programs of study page I am getting an accessibility error saying that the ID used for an element on the page is also used for another element on the page. As I'm looking at the code it references the program id for each program being used twice and that the ID's should be unique? Let me know if you need any more information.

Thanks!

![capture](https://user-images.githubusercontent.com/15264315/46148773-cb526f80-c236-11e8-863c-8c683eb58cc3.PNG)

**Comment by d4mation** [comment]
I just pushed a fix for this to Staging. Can you run your accessibility tester there and verify that the problem has been fixed, please? https://jacksoncollege.staging.wpengine.com/academics/programs-of-study/

Also, what tool do you use for testing accessibility? I use the one built into Google Chrome and I was able to see the same problem and confirmed that I was able to fix it, but I really like the way that yours highlights things on-screen.

**Comment by banksashleym** [comment]
Absolutely! Just checked and it did the trick!

We use a tool called SiteImprove. They have a single page checker as a [chrome extension](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl=en-US) and i liked it so much that we purchased the whole tool to be able to scan the entire site.

**Comment by d4mation** [comment]
This has been pushed Live 👍

**Comment by banksashleym** [comment]
Thank you!!
